src/content/getting_started/architecture/_index.en.md

```diff
@@ -3,11 +3,12 @@ title = "Architecture"
 date = 2020-02-19T21:00:30+01:00
 weight = 5
 +++
-
-The diagram below illustrates the basic architecture of Submariner:
-
-![Submariner Architecture](/images/submariner/architecture.jpg)
-
+<!-- markdownlint-disable line-length -->
+Submariner connects multiple Kubernetes clusters in a way that is secure and performant. Submariner flattens the networks between the
+connected clusters, and enables IP reachability between Pods, Services, and HostNetworks. Submariner also provides, via Lighthouse,
```

Comment:
@nyechiel HostNetworks can probably be confused to "Host to host connectivity between the clusters". As you know, we support Host to remoteService/Pod connectivity, but not Host to Host connectivity between the clusters.

Reply:
Ack, removing HostNetworks to keep it simple.